.hound.yml

```diff
@@ -0,0 +1,5 @@
+javascript:
+  enabled: false
+jshint:
+  enabled: false
+  config_file: .javascript-style.json
```

Comment:
If we're disabling this (`enabled: false`), do we need to commit the config file?